src/release-notes/v2021.06.24-1.md

```diff
@@ -0,0 +1,8 @@
+date: "2021-06-24"
+---
+version: "v2021.06.24-1"
+weight: 202106241
+---
```

Comment:
```suggestion
---
date: "2021-06-24"
version: "v2021.06.24-1"
weight: 202106241
---
```

Reply:
to match [the others](https://raw.githubusercontent.com/replicatedhq/kurl.sh/master/src/release-notes/v2021.06.24-0.md), since this netlify build failed